backend/tests/modules/project_rewards/test_rewards.py

```diff
@@ -53,6 +49,13 @@ def before(
     mock_graphql(mocker, epochs_events=[epoch])
 
 
+@pytest.fixture(scope="function")
+def mock_pending_epoch_snapshot_db(app, mock_users_db):
```

Comment:
I've double checked it and you're absolutely right. I wrote only a simple test for checking estimated_project_rewards with default values since they're currently validated in other tests, as you said :v: